pkgs/by-name/or/ory/package.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles, gcc }:
+
+buildGoModule rec {
+  pname = "ory";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "ory";
+    repo = "cli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-5N69/Gv4eYLbZNN+sEx+RcFyhGCT0hUxDCje1qrbWiY=";
+  };
+
+  buildInputs = [
+    gcc
+  ];
+
```

Comment:
buildGoModule still uses stdenv, so gcc & co get pulled in by default. This can be omitted.